src/background/actions/dialog-actions.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AsyncAction } from 'common/flux/async-action';
+
+export class DialogActions {
```

Comment:
Per offline discussion with @waabid, ended up refactoring this PR completely to use a new view-side store instead of a background store. This also removes all the generic-sounding `Dialog-` objects.

I considered using `IssueFilingActionMessageCreator` to trigger the open/close dialog actions, but it looks like the existing openDialog function isn't called anywhere, and it seems odd for something called a MessageCreator to have functions that invoke actions without using messages. I created a separate `CardsViewController` to wrap `CardsViewActions` instead.